src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const sett = new Set();
+  const copyy = {};
+
+  for (const x in robot) {
+    sett.add(robot[x]);
+  }
+
+  if (sett.size !== Object.keys(robot).length) {
+    return null;
```

Comment:
I think using Set() and this check is redundant. You can check if there is an existing key while adding a key.